app/src/main/java/com/randomappsinc/studentpicker/presentation/PresentationManager.kt

```diff
@@ -0,0 +1,98 @@
+package com.randomappsinc.studentpicker.presentation
+
+import android.animation.Animator
+import android.animation.AnimatorSet
+import android.animation.ObjectAnimator
+import android.content.Context
+import android.content.res.AssetFileDescriptor
+import android.media.AudioManager
+import android.media.MediaPlayer
+import android.os.Handler
+import android.view.View
+import android.view.animation.AccelerateInterpolator
+import android.view.animation.DecelerateInterpolator
+import com.randomappsinc.studentpicker.R
+import com.randomappsinc.studentpicker.utils.UIUtils
+
+/** The amount of time you need to wait for the drumroll crash to happen*/
```

Comment:
nit: Is it not possible to have this within the class?
```suggestion
/** The amount of time you need to wait for the drumroll crash to happen */
```

Reply:
Hmmm, we can put it inside a companion object but This is much less clean because under the hood when the bytecode is generated, a useless object is created. we can discuss more about it.